### PR TITLE
Update prices_ethereum_tokens.sql

### PR DESCRIPTION
**Pull request by 0xpibbles**

### CODE REVIEW

**Comment by jeff-dude** [comment]
hi @0xpibbles welcome to spellbook!

as the bot mentioned above, you will need to copy/paste the sentence to sign the license agreement. please comment only that single line 👍

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 1]
```diff
@@ -1621,7 +1621,9 @@ FROM
     ('gnx-genaro-network', 'ethereum', 'GNX', 0x6ec8a24cabdc339a06a172f8223ea557055adaa5, 9),
     ('pndc-pond-coin', 'ethereum', 'PNDC', 0x423f4e6138e475d85cf7ea071ac92097ed631eea, 18),
     ('crvusd-curvefi-usd-stablecoin', 'ethereum', 'CRVUSD', 0xf939e0a03fb07f59a73314e73794be0e57ac1b4e, 18),
-    ('mav-maverick-token', 'ethereum', 'MAV', 0x7448c7456a97769f6cd04f1e83a4a23ccdc46abd, 18)
+    ('mav-maverick-token', 'ethereum', 'MAV', 0x7448c7456a97769f6cd04f1e83a4a23ccdc46abd, 18),
+        ('frxeth-frax-token', 'ethereum', 'frxETH', 0x5e8422345238f34275888049021821e8e08caa1f, 18),
```
Comment (jeff-dude) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1625]: based on the gh action CI test failure, it looks like this token is invalid on our price provider API. please see returned result:
https://api.coinpaprika.com/v1/coins/frxeth-frax-token

you can remove this line here if you want to expedite the merge

**Review by jeff-dude — CHANGES_REQUESTED** [review]

**Review by 0xpibbles — COMMENTED** [review]
I have read the CLA Document and I hereby sign the CLA

**Comment by jeff-dude** [comment]
hmm, it didn't work. i think you need to leave a regular comment on the PR, rather than a review comment

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 2]
```diff
@@ -1642,3 +1644,5 @@ where contract_address not in (
     ,0x22acaee85ddb83a3a33b7f0928a0e2c3bfdb6a4f   -- PRXY
     ,0xe94b97b6b43639e238c851a7e693f50033efd75c   -- RNBW 
 )
+
+I have read the CLA Document and I hereby sign the CLA
```
Comment (jeff-dude) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1648]: @0xpibbles sorry for the confusion. this should not be in the code itself. please do remove here

you need to copy/paste:
I have read the CLA Document and I hereby sign the CLA

directly into a gh comment on the PR, at the bottom of the conversation page

**Comment by 0xpibbles** [comment]
I have read the CLA Document and I hereby sign the CLA

File: models/prices/ethereum/prices_ethereum_tokens.sql (hunk 2 above)
Comment (0xpibbles) [reply models/prices/ethereum/prices_ethereum_tokens.sql]: ahh i see, thanks for your patience and helping me out!

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 3]
```diff
@@ -1621,7 +1621,9 @@ FROM
     ('gnx-genaro-network', 'ethereum', 'GNX', 0x6ec8a24cabdc339a06a172f8223ea557055adaa5, 9),
     ('pndc-pond-coin', 'ethereum', 'PNDC', 0x423f4e6138e475d85cf7ea071ac92097ed631eea, 18),
     ('crvusd-curvefi-usd-stablecoin', 'ethereum', 'CRVUSD', 0xf939e0a03fb07f59a73314e73794be0e57ac1b4e, 18),
-    ('mav-maverick-token', 'ethereum', 'MAV', 0x7448c7456a97769f6cd04f1e83a4a23ccdc46abd, 18)
+    ('mav-maverick-token', 'ethereum', 'MAV', 0x7448c7456a97769f6cd04f1e83a4a23ccdc46abd, 18),
+    ('frxeth-frax-ether', 'ethereum', 'frxETH', 0x5e8422345238f34275888049021821e8e08caa1f, 18),
+    ('sdt-stakedao-token', 'ethereum', 'SDT', 0x73968b9a57c6e53d41345fd57a6e6ae27d6cdb2f, 18)
```
Comment (jeff-dude) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1626]: @0xpibbles it looks like this is an invalid id:
https://api.coinpaprika.com/v1/coins/sdt-stakedao-token

**Review by jeff-dude — CHANGES_REQUESTED** [review]

**Comment by jeff-dude** [comment]
closing due to inactivty, please do resubmit new prices when needed 👍

**Pull request closed without merging**